Cargo.lock

```diff
@@ -2759,9 +2759,9 @@ dependencies = [
 
 [[package]]
 name = "regex"
-version = "1.3.9"
+version = "1.4.0"
```

Comment:
I don't think you meant to update this file - can you change rustdoc to use `regex = "1"` instead?

Reply:
Good point.